client/src/components/AddDocumentForm.js

```diff
@@ -9,30 +10,37 @@ class AddDocumentForm extends Component {
     this.state = {
       fromLanguage: "",
       toLanguage: "",
-      haveAllFielsValue: false,
+      haveAllFieldsValue: false,
       dueDate: "",
-      text: ""
+      text: "",
+      name: ""
     };
   }
 
   handleSubmit = () => {
-    console.log(this.state);
+    postDocument();
```

Comment:
This can be removed from this PR, I think it will be added in a different ticket.

Reply:
I removed console.log from both files but i keep the post document function as i will be working with this function in my new ticket.